test/roby_app/test_plugin.rb

```diff
@@ -212,6 +212,22 @@ def perform_app_assertion(result)
                     perform_app_assertion interface.unit_tests.orogen_deployment_exists?
                 end
             end
+
+            describe "Temporary Root CA" do
+                before do
+                    Plugin.start_tmp_root_ca
+                end
+
+                after do
+                    Plugin.stop_tmp_root_ca
+                end
+                
+                it "has TmpRootCA been initialized" do
+                    assert_instance_of TmpRootCA, Plugin.has_tmp_root_ca?
+                end
+                
+            end
+
```

Comment:
The test you can already write is (pseudocode):

~~~
root_ca = create_tmp_root_ca
start_log_transfer_server(root_ca)
# Do a transfer using send_file_transfer_command
# verify that the file transfer completes
~~~